server/setup_server.sh

```diff
@@ -67,8 +67,8 @@ else
     pm2 startup systemd
 
     # ensure that a new process is started or that a running process is restarted
-    pm2 stop server.js
-    pm2 start server.js
-
+    pm2 stop ecosystem.config.js --env production
```

Comment:
ecosystem.js allows you to configure some more aspects of how it runs, and to give it whatever name you want on the pm2 status page (ie instead of simply being called server.js it'll be called BraveServer) among other things. I don't think it's making much of a difference in practice other than being more configurable/feature-rich way of doing the same thing.

I feel like sometimes we run the server.js file on its own in development, but this file is for using pm2 and deploying on a server and assumes a 'production' environment (even if its on a dev server for example, the 'production' environment is what tells the code to use the ssl certificates and https for example).